install/Knative-with-ICP.md

```diff
@@ -0,0 +1,158 @@
+# Knative Install on IBM Cloud Private
```

Comment:
@zxDiscovery  I still recommend building some numbered steps into this guide to make it easier for users to follow. Right now, this would be the only install guide that doesn't break down its sections further into numbered steps. For example, take a look at the other install guide for IBM, the one for IKS: https://github.com/knative/docs/blob/master/install/Knative-with-IKS.md

Reply:
This is close, but for consistency's sake, let's take the numbers out of the headings where you've added them, and add them to the section that doesn't have any, "Installing Knative Serving." @zxDiscovery